examples/compute_kinematics.py

```diff
@@ -0,0 +1,439 @@
+"""
+Compute kinematics
+============================
+
+Compute displacement, velocity and acceleration data on an example dataset.
+"""
+
+# %%
+# Imports
+# -------
+
+import numpy as np
+
+# For interactive plots: install ipympl with `pip install ipympl` and uncomment
+# the following line in your notebook
+# %matplotlib widget
+# Install circle_fit in your virtual environment with `pip install circle-fit`
+from circle_fit import taubinSVD
+from circle_fit.circle_fit import convert_input
+from matplotlib import pyplot as plt
+
+import movement.analysis.kinematics as kin
+from movement import sample_data
+from movement.io import load_poses
+
+# %%
+# Load sample dataset
+# ------------------------
+# First, we load an example dataset. In this case, we select the
+# `SLEAP_three-mice_Aeon_proofread` sample data.
+file_path = sample_data.fetch_sample_data_path(
+    "SLEAP_three-mice_Aeon_proofread.analysis.h5"
+)
```

Comment:
You can leave this as is, or you may also use the more direct `fetch_sample_data` method. In the first example, I used the one with the path because I wanted to showcase loading from files. That said, reminding people how to load from a file is not a bad thing (even if its a bit of duplication).